schema/testdata.go

```diff
@@ -28,6 +28,7 @@ type TestSourceOptions struct {
 	SkipTimes      bool // time of day types
 	SkipLargeTypes bool // e.g. large binary, large string
 	TimePrecision  time.Duration
+	SkipDecimals   bool
```

Comment:
Also, it's not shown on the diff for some reason (probably since the PR needs a rebase), but `SkipMaps` is not used either and we don't add the maps at all